state_machine/flight_manager.py

```diff
@@ -20,26 +20,29 @@ class FlightManager:
     -------
     __init__(self) -> None
         Initialize a flight manager object.
-    start_manager() -> None
-        Test running the state machine in a separate process.
-    start_state_machine(drone: Drone) -> None
-        Create and start a state machine in the event loop. This method should
-        be called in its own process.
-    start_kill_switch(state_machine_process: Process, drone: Drone) -> None
-        Create and start a kill switch in the event loop.
-    kill_switch(state_machine_process: Process, drone: Drone) -> Awaitable[None]
-        Enable the kill switch and wait until it activates. The drone should be
-        in manual mode after this method returns.
-    graceful_exit(drone: Drone) -> Awaitable[None]
+    start_manager()
+        Starts the state machine and kill switch in separate processes.
```

Comment:
This doc string was changed while merging accidently, change it back to how it was before

Reply:
Look through the commit history on this PR to find the old docstring. Or, if you need the practice. rewrite the docstring yourselves. Don't forget the documentation website if you have any questions about formatting.